tests/runkola

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+# Generate a new qemu image and run tests
+top=$(git rev-parse --show-toplevel)
+cd ${top}
+make
+cosa build-fast
```

Comment:
I feel like `cosa build-fast` should print a banner or something to make it clear what the caveats are. Or if we don't want to put it there, then at least in higher-level hacking scripts like this one which use it. Otherwise, I can definitely see OSTree developers in the future who might not be familiar with the underlying tools banging their head trying to figure out why e.g. their initramfs-related change isn't being picked up.

Anyway, don't have to address this right away.